system-test/bigtable.js

```diff
@@ -374,6 +374,42 @@ describe('Bigtable', function() {
     });
   });
 
+  describe('consistency tokens', function() {
+    it('should generate consistency token', done => {
+      TABLE.generateConsistencyToken(function(err, token) {
+        assert.ifError(err);
+        assert.strictEqual(typeof token, 'string');
+        done();
+      });
+    });
+
+    it('should return error for checkConsistency of invalid token', done => {
+      TABLE.checkConsistency('dummy-token', function(err, res) {
+        assert.strictEqual(err.message, '3 INVALID_ARGUMENT: Invalid token');
```

Comment:
Knowing our system test flakiness history too well, this is bound to be broken someday by an upstream change somewhere. To be a bit more future-proof, we could either check just the error code, or that the message includes simply "invalid token".